jekyll/_cci2/api_v2_introduction.md

```diff
@@ -0,0 +1,226 @@
+
+- Retrieve a workflow by ID
+- Trigger Pipelines with parameters
+- Trigger particular workflows within Pipelines
+- Retrieve a Pipeline by ID
+- Retrieve recent Pipelines for an organization or for a project (ideally with branch filtering)
+
+**Note** In API v1.1, the term "build processing" was used to describe how projects were built and processed; however, with the API v2 release, "build processing" has been replaced by "pipelines." New projects will have "pipelines" set by default, whereas existing projects will not be changed.
+
+## Getting Started with the API v2
+
+The v2 API is very similar to the current v1.1 API with some notable exceptions.
+
+Here is a simple example using `curl` to trigger a pipeline with parameters 
+
+```
+curl -u ${CIRCLECI_TOKEN}: -X POST --header "Content-Type: application/json" -d '{
+  "parameters": {
+    "myparam": "./myspecialdir",
+    "myspecialversion": "4.8.2"
+  }
+}' https://circleci.com/api/v2/project/${project_slug}/pipeline
+```
+
+In the above example the `project_slug` would take the form :vcs/:org/:project. For example, the project slug `gh/CircleCI-Public/circleci-cli` tells `CircleCI` to use the project found in the GitHub organization CircleCI-Public in the repository named `circleci-cli`.
+
+### Authentication
+
+The CircleCI API v2 enables users to be authenticated in much the same way as they would be authenticated in the previous v1.1 API. A simple way to authenticate is to send your API token as the username of the HTTP request. For example, if you have set `CIRCLECI_TOKEN` in your shell's environment, then you could then use `curl` with that token like the example shown below:
+
+`curl -u ${CIRCLECI_TOKEN} https://circleci.com/api/v2/me`
+
+For more detailed information on how to authenticate with a token, please refer to the [Authentication](https://circleci.com/docs/api/#authentication) section in the main CirlceCI API documentation.
+
+### Pipelines
+
+#### What is a Pipeline?
+
+Pipelines are a CircleCI designation for the full set of configurations you run when triggering work on your projects on the CircleCI platform. Pipelines contain your workflows, which in turn coordinate your jobs. 
+
+#### What Are the Practical Implications of Pipelines?
+
+In this new approach, jobs running a configuration prior to 2.1 will run in a workflow. Jobs running a 2.1 configuration will require workflows to be defined before the job may be run. In the future, CircleCI will add a new pipeline-level parameters clause in `config.yml`, referenced in the `pipeline` scope, as well as launch a new version of the CircleCI API that has the ability to trigger pipelines with parameters and retrieve them and their workflows. In addition to these changes, CircleCI is also creating a new UI that will unify the experience of viewing your pipelines and their jobs coordinated in workflows.
```

Comment:
This paragraph should probably be deleted -- if you keep it, let's talk about it because it has some inaccuracies.

Reply:
I went ahead and deleted. It is not really necessary for this first iteration of the Introduction.